recipe_sha: 29d0da4ec534ab1bb22cd54ae0e9717543925684dc6389a310402b3186f24bf3  # The hash of the recipe's meta.yaml at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |-
  2024-06-04T15:44:48 linux-aarch64 Writing pre-patch repodata
  2024-06-04T15:44:48 linux-aarch64 Applying patch instructions
  2024-06-04T15:44:48 linux-aarch64 Writing patched repodata
  2024-06-04T15:44:48 linux-aarch64 Building current_repodata subset
  2024-06-04T15:44:48 linux-aarch64 Writing current_repodata subset
  2024-06-04T15:44:48 linux-aarch64 Writing index HTML
  2024-06-04T15:44:48 Subdir: noarch Gathering repodata
  2024-06-04T15:44:48 noarch Writing pre-patch repodata
  2024-06-04T15:44:48 noarch Applying patch instructions
  2024-06-04T15:44:48 noarch Writing patched repodata
  2024-06-04T15:44:48 noarch Building current_repodata subset
  2024-06-04T15:44:48 noarch Writing current_repodata subset
  2024-06-04T15:44:48 noarch Writing index HTML
  /opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py:501: UserWarning: RECIPE_PATH received is a file (/opt/recipe/meta.yaml).
  It should be a path to a folder.
  Forcing conda-build to use the recipe file.
    warnings.warn(
  WARNING: No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.22
  Adding in variants from internal_defaults
  Adding in variants from /opt/host-conda-bld/conda_build_config_0_-e_conda_build_config.yaml
  Adding in variants from /opt/host-conda-bld/conda_build_config_1_-e_bioconda_utils-conda_build_config.yaml
  Skipped: ragout from /opt/recipe defines build/skip for this configuration ({'cxx_compiler_version': '12', 'channel_targets': 'bioconda main', 'target_platform': 'linux-64', 'CONDA_BUILD_SYSROOT': '', 'cxx_compiler': 'gxx', 'python': '3.11.* *_cpython'}).
  Renaming work directory '/opt/conda/conda-bld/ragout_1717515889209/work' to '/opt/conda/conda-bld/ragout_1717515889209/work_moved_ragout-2.3-py311h2a4ad6c_7_linux-64_main_build_loop'
  shutil.move(work)=/opt/conda/conda-bld/ragout_1717515889209/work, dest=/opt/conda/conda-bld/ragout_1717515889209/work_moved_ragout-2.3-py311h2a4ad6c_7_linux-64_main_build_loop)
  Skipped: ragout from /opt/recipe defines build/skip for this configuration ({'cxx_compiler_version': '12', 'channel_targets': 'bioconda main', 'target_platform': 'linux-64', 'CONDA_BUILD_SYSROOT': '', 'cxx_compiler': 'gxx', 'python': '3.12.* *_cpython'}).
  Renaming work directory '/opt/conda/conda-bld/ragout_1717515889209/work' to '/opt/conda/conda-bld/ragout_1717515889209/work_moved_ragout-2.3-py312h28adbb1_7_linux-64_main_build_loop'
  shutil.move(work)=/opt/conda/conda-bld/ragout_1717515889209/work, dest=/opt/conda/conda-bld/ragout_1717515889209/work_moved_ragout-2.3-py312h28adbb1_7_linux-64_main_build_loop)
  Skipped: ragout from /opt/recipe defines build/skip for this configuration ({'cxx_compiler_version': '12', 'channel_targets': 'bioconda main', 'target_platform': 'linux-64', 'CONDA_BUILD_SYSROOT': '', 'cxx_compiler': 'gxx', 'python': '3.9.* *_cpython'}).
  Renaming work directory '/opt/conda/conda-bld/ragout_1717515889209/work' to '/opt/conda/conda-bld/ragout_1717515889209/work_moved_ragout-2.3-py39h9e0f934_7_linux-64_main_build_loop'
  shutil.move(work)=/opt/conda/conda-bld/ragout_1717515889209/work, dest=/opt/conda/conda-bld/ragout_1717515889209/work_moved_ragout-2.3-py39h9e0f934_7_linux-64_main_build_loop)
  Attempting to finalize metadata for ragout
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  BUILD START: ['ragout-2.3-py38h2123bcc_7.tar.bz2']
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Solving environment (_h_env): ...working... done

  ## Package Plan ##

    environment location: /opt/conda/conda-bld/ragout_1717515889209/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh


  The following NEW packages will be INSTALLED:

      _libgcc_mutex:    0.1-conda_forge           conda-forge
      _openmp_mutex:    4.5-2_gnu                 conda-forge
      bzip2:            1.0.8-hd590300_5          conda-forge
      ca-certificates:  2024.6.2-hbcca054_0       conda-forge
      ld_impl_linux-64: 2.40-hf3520f5_1           conda-forge
      libffi:           3.4.2-h7f98852_5          conda-forge
      libgcc-ng:        13.2.0-h77fa898_7         conda-forge
      libgomp:          13.2.0-h77fa898_7         conda-forge
      libnsl:           2.0.1-hd590300_0          conda-forge
      libsqlite:        3.45.3-h2797004_0         conda-forge
      libstdcxx-ng:     13.2.0-hc0a3c3a_7         conda-forge
      libuuid:          2.38.1-h0b41bf4_0         conda-forge
      libxcrypt:        4.4.36-hd590300_1         conda-forge
      libzlib:          1.3.1-h4ab18f5_1          conda-forge
      ncurses:          6.5-h59595ed_0            conda-forge
      openssl:          3.3.0-h4ab18f5_3          conda-forge
      pip:              24.0-pyhd8ed1ab_0         conda-forge
      python:           3.8.19-hd12c33a_0_cpython conda-forge
      readline:         8.2-h8228510_1            conda-forge
      setuptools:       70.0.0-pyhd8ed1ab_0       conda-forge
      tk:               8.6.13-noxft_h4845f30_101 conda-forge
      wheel:            0.43.0-pyhd8ed1ab_1       conda-forge
      xz:               5.2.6-h166bdaf_0          conda-forge

  Preparing transaction: ...working... done
  Verifying transaction: ...working... done
  Executing transaction: ...working... done
  ERROR conda.core.link:_execute(950): An error occurred while installing package 'conda-forge::libstdcxx-ng-13.2.0-hc0a3c3a_7'.
  Rolling back transaction: ...working... done
  Traceback (most recent call last):
    File "/opt/conda/bin/conda-build", line 11, in <module>
      sys.exit(execute())
    File "/opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 590, in execute
      api.build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 250, in build
      return build_tree(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3638, in build_tree
      packages_from_this = build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2409, in build
      create_build_envs(top_level_pkg, notest)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2183, in create_build_envs
      environ.create_env(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/environ.py", line 1049, in create_env
      _execute_actions(prefix, precs)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/environ.py", line 1334, in _execute_actions
      unlink_link_transaction.execute()
    File "/opt/conda/lib/python3.10/site-packages/conda/core/link.py", line 354, in execute
      self._execute(
    File "/opt/conda/lib/python3.10/site-packages/conda/core/link.py", line 970, in _execute
      raise CondaMultiError(
  conda.CondaMultiError: [Errno 28] No space left on device
  ()
# Last 100 lines of the build log.
